File: clients/async-scala/src/main/scala/io/swagger/client/model/Stats.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class Stats (
  rootSymbol: String,
currency: Option[String],
volume24h: Option[Number],
turnover24h: Option[Number],
openInterest: Option[Number],
openValue: Option[Number]
)

File: clients/typescript-angular/API/Client/Stats.ts
/// <reference path="api.d.ts" />

namespace API.Client {
    'use strict';

    export interface Stats {

        rootSymbol: string;

        currency?: string;

        volume24h?: number;

        turnover24h?: number;

        openInterest?: number;

        openValue?: number;
    }

}
